packages/rum-core/src/performance-monitoring/transaction-service.js

```diff
@@ -111,7 +111,6 @@ class TransactionService {
         pageLoadTraceId: config.pageLoadTraceId,
         pageLoadSampled: config.pageLoadSampled,
         pageLoadSpanId: config.pageLoadSpanId,
-        pageLoadTransactionName: config.pageLoadTransactionName,
```

Comment:
Why we don't need this here? We should still set the correct transaction name at the start of the transaction.
I think we discuss this previously, the transaction should have the correct name as soon as possible, e.g. in case we want to decide sampling based on the name.

Reply:
explained in the below comment.